pipelines/cflinuxfs4.yml

```diff
@@ -227,12 +232,40 @@ resources:
     secret_access_key: ((svc-buildpacks-aws-team-secret-key))
     assume_role_arn: ((svc-buildpacks-aws-team-assume-role-arn))
 
+- name: check-interval
+  type: cron
+  source:
+    expression: "32 09 * * *"
```

Comment:
Reduce from `0 * * * *` (i.e. on the hour, every hour) to a psudo-random time once per day